third_party/move/move-vm/runtime/src/native_functions.rs

```diff
@@ -81,8 +81,26 @@ impl NativeFunctions {
     where
         I: IntoIterator<Item = (AccountAddress, Identifier, Identifier, NativeFunction)>,
     {
+        let vector_bytecode_instruction_methods = HashSet::from([
+            "empty",
```

Comment:
Any reason for adding this check now (in this PR)?

Do we really want to prevent this? There's nothing wrong with having native functions that have the same functionality as instructions.

Reply:
previous comment:

> not sure if this check is even needed (this is mostly to make sure we don't create new native function with same name as bytecode instruction), or if it's better to remove it - or on the other hand panic here.
> 
> but this is basically to be a fixed replacement of the filter in aptos-move/aptos-vm/src/natives.rs
>

Reply:
I don't think there's a general reason to disallow these in MoveVM. I'd rather keep the filter in AptosVM. Is there any other reason for wanting to remove it from AptosVM?